lib/cext/include/ruby/intern.h

```diff
@@ -441,6 +441,7 @@ VALUE rb_eval_cmd(VALUE, VALUE, int);
 VALUE rb_eval_cmd_kw(VALUE, VALUE, int);
 int rb_obj_respond_to(VALUE, ID, int);
 int rb_respond_to(VALUE, ID);
+VALUE rb_method_to(VALUE, ID);
```

Comment:
What's this?

Reply:
Sorry, this is bad squash-rebase during git history cleanup >.<